components/FileTable.tsx

```diff
@@ -23,36 +27,84 @@ interface IFileDownloadModalProps {
     isOpen: boolean;
 }
 
-const FileDownloadModal: React.FunctionComponent<IFileDownloadModalProps> = (
+const CDSInstructions: React.FunctionComponent<{ files: Entity[] }> = (
+    props
+) => {
+    return (
+        <>
+            <p>
+                Your selection includes Level 1 and/or Level 2 sequencing data:
+            </p>
+            <pre className="pre-scrollable">
+                <code>
+                    {props.files.map((f) => getFileBase(f.filename)).join('\n')}
+                </code>
+            </pre>
+            <p>
+                These should be obtained through{' '}
+                <a
+                    href="https://datacommons.cancer.gov/repository/cancer-data-service"
+                    target="_blank"
+                >
+                    CDS/dbGaP
+                </a>
+                . Please refer to the documentation there on how to download the
+                data.
+            </p>
+        </>
+    );
+};
+
+const SynapseInstructions: React.FunctionComponent<{ files: Entity[] }> = (
     props
 ) => {
     const script = props.files
         .map((f) => `synapse get ${f.synapseId}`)
         .join('\n');
 
+    return (
+        <>
+            <p>
+                You can use the synapse command line client to download the
+                selected files:
+            </p>
+            <pre className="pre-scrollable">
+                <code>{script}</code>
+            </pre>
+            <p>
+                For more information see the{' '}
+                <a
+                    href="https://docs.synapse.org/articles/downloading_data.html"
+                    target="_blank"
+                >
+                    synapse documentation
+                </a>
+            </p>
+        </>
+    );
+};
+
+const FileDownloadModal: React.FunctionComponent<IFileDownloadModalProps> = (
+    props
+) => {
+    const cdsFiles = props.files.filter(
+        (f) => f.level === 'Level 1' || f.level === 'Level 2'
+    );
+    const synapseFiles = props.files.filter(
+        (f) => f.level !== 'Level 1' && f.level !== 'Level 2'
+    );
```

Comment:
@onursumer in theory there could be imaging level 1 data that you prolly would download from synapse (it's only the sequencing data that one would download from CDS). So any Assay type that's not "Imaging"

Reply:
@inodb thanks for the clarification!